contents/kotlin/kotlin.md

```diff
@@ -0,0 +1,87 @@
+# Introduction to Kotlin
+
+Author: [Shradheya Thakre](https://github.com/tshradheya)
+
+**Table of Contents**
+
+* [Overview](#overview)
+* [Why Kotlin?](#why-kotlin)
+    * [Concise](#concise)
+    * [Safe](#safe)
+    * [Interoperable](#interoperable)
+    * [Tool Friendly](#tool-friendly)
+* [Kotlin for Android Apps](#kotlin-for-android-apps)
+    * [Advantages of Shifting to Kotlin](#advantage-kotlin)
+    * [Drawbacks of Shifting to Kotlin](#drawbacks-kotlin)
+
+
+
+
+
+
+# Overview
+
+Kotlin is a general purpose, open source, statically typed “pragmatic” programming language for the JVM and Android that combines object-oriented and functional programming features.
```

Comment:
I've seen some websites using the word "pragmatic" as well, but I think it adds more confusion than explanation, especially with the " ". The sentence would work just as well without the word.

Reply:
Hmm, I thought the pragmatic makes a huge difference. Kotlin is so attractive because of the practical approach they took instead of trying to become a perfect language, theoretically sound or whatsoever.

Reply:
I get what you mean, but the overview should be a clear and concise introduction to the topic. Having the word "programatic" makes the user wonder what it means to be a "pragmatic programming language". So it takes attention away from what Kotlin is at a glance. If anything, this could be explained a little down in the main points, but in the overview it's a little too much to be added in. 

And also, the " " around the word doesn't help with clarifying the meaning of the word.

Reply:
As suggested by @Haozhe321 , its better to get rid of the word as it adds more confusion to people completely new to Kotlin

Reply:
I see the point but pragmatic is a common jargon (we can always not use the jargon) and is one of the biggest selling points of Kotlin.

By removing pragmatic, Kotlin now becomes general purpose and open source. So the overview is that it has not many differences with other languages? That's just not attractive.